core_perception/lidar_euclidean_cluster_detect/launch/lidar_euclidean_cluster_detect.launch

```diff
@@ -9,19 +9,15 @@
   <arg name="sync" default="false" />
   <arg name="use_diffnormals" default="false" />
   <arg name="pose_estimation" default="true" />
-  <arg name="clip_min_height" default="-1.3" />
-  <arg name="clip_max_height" default="0.5" />
-
   <arg name="keep_lanes" default="false" />
   <arg name="keep_lane_left_distance" default="5" />
   <arg name="keep_lane_right_distance" default="5" />
   <arg name="cluster_merge_threshold" default="1.5" />
   <arg name="clustering_distance" default="0.75" />
-
   <arg name="use_vector_map" default="false" />
   <arg name="wayarea_gridmap_layer" default="wayarea" />
 
-  <arg name="output_frame" default="velodyne" />
+  <arg name="calibration_params_file" default="$(find lidar_euclidean_cluster_detect)/config/calibration.yaml"/>
```

Comment:
use arg value here

Reply:
The above syntax is correct. Its not in the carmajava.